### ISSUE TITLE
Connecting to database verbose output occurs twice.

### ISSUE THREAD

**Issue by mibby** [Phoenix616/ChestShopNotifier#5]
From console / logs, connecting to database outputs twice. Not sure if intended or it spams connections until it connects?

```
[04:01:36] [Server thread/INFO]: [ChestShopNotifier] Enabling ChestShopNotifier v1.1.5
[04:01:36] [Server thread/INFO]: [ChestShopNotifier] Connecting to the database...
[04:01:36] [Server thread/INFO]: [ChestShopNotifier] Connecting to the database...
[04:01:36] [Server thread/INFO]: [ChestShopNotifier] Connected!
```

**Comment by Phoenix616** [comment]
Yeah, looks like it opens a connection twice, once in the updateConfiguration call [here](https://github.com/Phoenix616/ChestShopNotifier/blob/master/src/com/wfector/notifier/ChestShopNotifier.java#L64) and once 4 lines below again.

The whole database connection handling in general looks kinda wonky to me...

**Comment by mibby** [comment]
Any plans to rewrite it?

**Comment by mibby** [comment]
@Phoenix616 Any update to all the SQL errors? :/

**Comment by Phoenix616** [comment]
Sorry, I currently don't have any time to properly maintain this plugin further than to just keep it working. I'm open for PR's addressing this problem or the whole database situation 'though.

**Comment by mibby** [comment]
:( Well if you ever do find the time, I'll be here to help test all the SQL exceptions.

**Comment by Phoenix616** [comment]
Fixed with 93b9c2ba
